libraries/AP_Camera/AP_Camera_Backend.h

```diff
@@ -128,6 +131,13 @@ class AP_Camera_Backend
         uint32_t feedback_trigger_logged_count; // ID sequence number
     } camera_feedback;
 
+    // Picture settings
+    struct {
+        bool enabled;               // true if taking multiple pictures is enabled
+        float time_interval;        // time interval (in seconds) between two consecutive pictures
+        int16_t num_remaining;      // number of pictures still to be taken
```

Comment:
maybe put a comment here saying that -1 means, "take unlimited number of pictures"